packages/components/src/components/toast/toast.scss

```diff
@@ -33,6 +33,17 @@
             }
         }
     }
+    &__notification {
+        .dc-toast__message {
+            background: var(--general-active);
+            font-size: 1.2rem;
+            padding: 0.9rem 1.2rem;
+            /* iPhone SE screen height fixes due to UI space restrictions */
+            @media only screen and (max-height: 580px) {
+                font-size: 1rem;
+            }
```

Comment:
Font should be handled by `Text` component for both desktop and responsive.

Reply:
Will remove it